File: src/main/java/com/mhl/mycompanybackend/pojo/TaskRequest.java
package com.mhl.mycompanybackend.pojo;

import com.fasterxml.jackson.annotation.JsonFormat;

import java.util.Date;
import java.util.List;

public class TaskRequest {
    String task_name;
    Long creator_id;
    @JsonFormat(pattern = "yyyy-MM-dd")
    private Date creation_date;
    private String description;
    @JsonFormat(pattern = "yyyy-MM-dd")
    private Date deadline;
    Long executor_id;
    List<Long> members;

    public String getTask_name() {
        return task_name;
    }

    public void setTask_name(String task_name) {
        this.task_name = task_name;
    }

    public Date getCreation_date() {
        return creation_date;
    }

    public void setCreation_date(Date creation_date) {
        this.creation_date = creation_date;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Date getDeadline() {
        return deadline;
    }

    public void setDeadline(Date deadline) {
        this.deadline = deadline;
    }

    public Long getCreator_id() {
        return creator_id;
    }

    public void setCreator_id(Long creator_id) {
        this.creator_id = creator_id;
    }

    public Long getExecutor_id() {
        return executor_id;
    }

    public void setExecutor_id(Long executor_id) {
        this.executor_id = executor_id;
    }

    public List<Long> getMembers() {
        return members;
    }

    public void setMembers(List<Long> members) {
        this.members = members;
    }
}

File: src/main/java/com/mhl/mycompanybackend/pojo/TaskRequest.kt
package com.mhl.mycompanybackend.pojo

import com.fasterxml.jackson.annotation.JsonFormat
import java.util.*

class TaskRequest {
    @JvmField
    var task_name: String? = null
    @JvmField
    var creator_id: Long? = null

    @JvmField
    @JsonFormat(pattern = "yyyy-MM-dd")
    var creation_date: Date? = null
    @JvmField
    var description: String? = null

    @JvmField
    @JsonFormat(pattern = "yyyy-MM-dd")
    var deadline: Date? = null
    @JvmField
    var executor_id: Long? = null
    @JvmField
    var members: List<Long>? = null
}
